src/mpcCoreKit.ts

```diff
@@ -127,11 +131,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
 
     this.options = options as Web3AuthOptionsWithDefaults;
 
-    if (this.options.asyncStorageKey) {
-      this.currentStorage = AsyncStorage.getInstance(this._storageBaseKey, options.asyncStorageKey);
-    } else {
-      this.currentStorage = BrowserStorage.getInstance(this._storageBaseKey, this.options.storageKey);
-    }
+    this.currentStorage = new AsyncStorage(this._storageBaseKey, options.storage);
```

Comment:
Please define what steps dapp should follow to remain backward compatible in terms of storage, for ex: some app might already be using "local" storage key already and it will just break for existing users if they pass some other storage class here. So migration strategy should be very carefully defined.

Reply:
add documentation for migration 
https://github.com/Web3Auth/mpc-core-kit/blob/5338a1596042f5c3deb6830da98ef697a36f549e/src/interfaces.ts#L301